tests/test_serialization_options.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2021 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from test_helpers import execute_command
+
+
+@pytest.mark.parametrize("eval, max_object_depth, expected_result", [
```

Comment:
What do you think about adding an `ids=` param here to make the test function name more readable, as WPT does? https://github.com/web-platform-tests/wpt/blob/6ba9288e50a8881a791b3d2f7814df799846658e/webdriver/tests/print/orientation.py#L17

Docs: https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids

> If you leave it up to pytest to generate ids, you will see that most of them are pretty good, but others feel somewhat random. The reason being that non-primitive types such as dict, list, tuple or instances of classes are non trivial to translate into ids.

(source: https://hackebrot.github.io/pytest-tricks/param_id_func/)

Reply:
done